readme.md

```diff
@@ -44,25 +44,120 @@ A helper function for constructing native Pact commands.
+
+Pact.lang.mkMeta(<sender:string> , <chainId:string>, <gasPrice: nunmber>, <gasLimit: number>) -> <meta: object>
+  ex: mkMeta("Bob", "Chain-1", 20, 30) -> { "sender": "Bob", "ChainId": "Chain-1", "gasPrice": 20, "gasLimit": 30 }
 ```
 
 NB: `JSON.stringify`, which is used here, generally converts floating point numbers correctly but fails for high precision scientific numbers < 0; you will need to manually convert them.
 e.g. `JSON.stringify(.0000001) -> '1e-7'` is incorrect as Pact has infinite precision decimals but does not interpret scientific numbers, so the proper conversion would be `JSON.stringify(.0000001) -> '0.0000001'`
 
 
-### Simple API
+
+### Simple API Fetch
+
+Simple fetch functions to make API request to a running Pact Server and retrieve the results.
+
+```
+* A Command Object to Execute in send or local.
+* @typedef {Object} execCmd
+* @property pactCode {string} - pact code to execute
+* @property keyPairs {array or object} - array or single ED25519 keypair
+* @property nonce {string} - nonce value, default at current time
+* @property envData {object} - JSON message data for command, default at empty obj
+* @property meta {object} - meta information, see mkMeta
+*/
+```
+```
+## Make API request to execute a command or commands in the public server and retrieve request keys of the txs.
+
+Pact.fetch.send([<execCmd:object>], <apiHost:string>) -> {"requestKeys": [...]}
+
+  ex:
+    const cmds = [{
+                    keyPairs: KEY_PAIR,
+                    pactCode: "todos.delete-todos "id-1""
+                  },{
+                    keyPairs: KEY_PAIR,
+                    pactCode: "todos.delete-todos "id-2""
+                  },{
+                    keyPairs: KEY_PAIR,
+                    pactCode: "todos.delete-todos "id-3""
+                  }]
+
+    Pact.fetch.send(cmds, API_HOST) -> { requestKeys: [
+                                          "6ue-lrwXaLcDyxDwJ1nuLzOfFtnQ2TaF0_Or_X0KnbE",
+                                          "P7qDsrt3evfEjtlQAW_b1ZPS7LpAZynCO8wx99hc5i0",
+                                          "qqhiEAuerIBrkZArSXPZxybQLzkTzHcwiB4ZrRU7FJM" ]}
+```
+```
+## Make API request to execute a single command in the local server and retrieve the result of the tx. (Used to execute commands that read DB)
+Pact.fetch.local(<execCmd:object>, <apiHost:string>) -> {result}
+
+  ex:     
+    const cmd = {
+        keyPairs: KEY_PAIR,
+        pactCode: `(todos.read-todos)`
+      };
+
+    Pact.fetch.local(cmd, API_HOST) ->
+    { status: "success",
+       data: [{ id: "id-1"
+                title: "wash"
+                completed: false
+              }]
+    }
+```
+```
+## Make API request to retrieve result of a tx or multiple tx's with request keys.
+Pact.fetch.poll({requestKeys: ["..."]}, <apiHost:string>) -> [{requestKey: "...", result: {...}}, ...]
+
+  ex:
+    const cmd = { requestKeys: [ "6ue-lrwXaLcDyxDwJ1nuLzOfFtnQ2TaF0_Or_X0KnbE",
```

Comment:
It might be better to show an example of using request keys from a previously sent command(s)

Reply:
Used request keys from "Pact.fetch.send" example.